src/sites/twitch-twilight/modules/chat/index.js

```diff
@@ -425,6 +431,59 @@ export default class ChatHook extends Module {
 			for(const inst of instances)
 				this.fixPinnedCheer(inst);
 		});
+
+		this.TabComplete.ready((cls, instances) => {
+			for(const inst of instances)
+				this.setupTabCompletion(inst);
+		});
+		this.TabComplete.on('mount', this.setupTabCompletion, this);
+	}
+
+	getResultsForSets(input, userID, channelID, renderEmoteSuggestion) { // eslint-disable-line
+		const search = input.substring(1),
```

Comment:
What's the casing on this? Do we need to toLowerCase it for proper checking against the lower-case emote names?